content/en/docs/reference/using-api/api-overview.md

```diff
@@ -84,7 +84,7 @@ Currently, there are several API groups in use:
 *  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION`, and use `apiVersion: $GROUP_NAME/$VERSION`
    (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in [Kubernetes API reference](/docs/reference/).
 
-The two paths that support extending the API with [custom resources](/docs/concepts/api-extension/custom-resources/) are:
+The two paths that support extending the API with [custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
 
  - [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
```

Comment:
Does this make more sense:
```suggestion
The two paths that support [extending the API](/docs/concepts/extend-kubernetes/api-extension/) are:

 - {{< glossary_tooltip term_id="CustomResourceDefinition" text="CustomResourceDefinition" >}}
```
?

Reply:
no, as I replied above, I hate glossary tool tips and I think we are abusing it today.